src/main/java/org/myrobotlab/service/interfaces/NeoPixel2Control.java

```diff
@@ -0,0 +1,75 @@
+/**
+ *                    
+ * @author Christian Beliveau
+ *  
+ * This file is part of MyRobotLab (http://myrobotlab.org).
+ *
+ * MyRobotLab is free software: you can redistribute it and/or modify
+ * it under the terms of the Apache License 2.0 as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version (subject to the "Classpath" exception
+ * as provided in the LICENSE.txt file that accompanied this code).
+ *
+ * MyRobotLab is distributed in the hope that it will be useful or fun,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * Apache License 2.0 for more details.
+ *
+ * All libraries in thirdParty bundle are subject to their own license
+ * requirements - please refer to http://myrobotlab.org/libraries for 
+ * details.
+ * 
+ * Enjoy !
+ * 
+ * */
+
+package org.myrobotlab.service.interfaces;
+
+import org.myrobotlab.framework.interfaces.NameProvider;
+
+public interface NeoPixel2Control extends NameProvider {

```

Comment:
Why a new interface?

Reply:
because its different than NeoPixelControl..

it has clear() which NeoPixelControl didn't support
Additionally it has fully explicit typed values for 
public void attachNeoPixel2Controller(NeoPixel2Controller controller);
public void detachNeoPixel2Controller(NeoPixel2Controller controller);

and it does not have any "helper" attach methods which have additional parameters.